src/test/shared/credentials/loginManager.test.ts

```diff
@@ -66,7 +65,18 @@ describe('LoginManager', async function () {
     })
 
     function assertTelemetry(expected: Parameters<typeof recordAwsValidateCredentials>[0]): void {
-        assert.deepStrictEqual(recordAwsValidateCredentialsSpy.args[0][0], expected)
+        const passive = expected.passive
```

Comment:
Passive is outside of metadata which is unfortunate, though makes sense as it is attached to all metrics. So I'll need to make a better way to test for it.